app/src/main/res/drawable/continent_africa.xml

```diff
@@ -0,0 +1,14 @@

```

Comment:
Could you please simplify this massive path, remove the translation group around it and use the proper heigth/width values already here? This is best done in Inkscape before creating the XML.

Reply:
Please do the same for the other new drawables.

Reply:
> Could you please simplify this massive path

How much more do I need to simplify? I've already simplified all them down to ~130 points.

> remove the translation group around it and use the proper heigth/width values already here

Could you please give me a hint on how to do this with Inkscape? 🤔

Reply:
In Inkscape, just select the group and ungroup it. in the status bar, it will tell you whether you selected a group or a path. Just ungroup the groups until they are gone.

Reply:
Ok, the problem was the viewbox, I've removed it and simplified (a lot) the svg graphics. But I wasn't able to simplify more the actual path, within Inkscape it only lost the form but didn't reduce the number of points.